src/__tests__/index.test.ts

```diff
@@ -31,160 +51,200 @@ afterAll(() => {
   axiosCachingDns.reset();
 });
 
-test('query google with baseURL and relative url', async () => {
-  axiosCachingDns.registerInterceptor(axios);
-
-  const { data } = await axios.get('/finance', {
-    baseURL: 'http://www.google.com',
-    // headers: { Authorization: `Basic ${basicauth}` },
+describe("LRU Cache Tests", () => {
```

Comment:
This next bit looks confusing but it's essentially just creating two sets of tests (one for `lru-cache` and one for `redis`). The same test cases will be run on each.

The only difference is we're using `beforeAll` to set the `useRedis` parameter and we're clearing the cache stats before each test group